src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -672,6 +673,23 @@ Namespace Microsoft.CodeAnalysis.Operations
             Return New InvalidOperation(children, _semanticModel, syntax, type, constantValue, isImplicit)
         End Function
 
+        Private Function CreateBoundAttributeOperation(boundAttribute As BoundAttribute) As IAttributeOperation
+            Dim isAttributeImplicit = boundAttribute.WasCompilerGenerated
+            If boundAttribute.HasErrors OrElse boundAttribute.ConstructorArguments.Length <> boundAttribute.Constructor.ParameterCount Then
```

Comment:
Bailing if there are any errors doesn't seem correct. We should only be bailing if it was so broken we can't make an IObjectCreationOperation.